build/helper/helper.py

```diff
@@ -47,15 +46,29 @@ def add_to_path(p):
         sys.path = old_path
 
 
+type_mapping = {
```

Comment:
move this to inside the function, and rename to type_map (noun)

Reply:
Done